src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -88,7 +88,8 @@ export class InputActionDialog {
   }
 
   checkIfJenkinsIsUpAndProceedURL() {
-    let url = 'https://jenkins.openshift.io';
+    let url = 'https://jenkins.prod-preview.openshift.io';
```

Comment:
We need to set the URL via the env var that are set with SaaSHearder in https://github.com/openshiftio/saas-openshiftio/blob/master/dsaas-services/fabric8-ui.yaml#L10
and referenced like https://github.com/fabric8-ui/fabric8-ui/blob/master/openshift/fabric8-ui.app.yaml#L57

Reply:
Looked everywhere but didn't find where we are setting the jenkins proxy url.
What we found is that `this.build.jenkinsBuildURL` in the current context has the right jenkins proxy url which can be hit to un-idle.


```
let url = this.build.jenkinsBuildURL + '?token_json=' + JSON.stringify(token_json);
``` 
is being used to call the jenkins proxy with the user's token - this will trigger un-idling.